server/resp/src/main/java/org/infinispan/server/resp/commands/hll/PFADD.java

```diff
@@ -0,0 +1,76 @@
+package org.infinispan.server.resp.commands.hll;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.AdvancedCache;
+import org.infinispan.commons.dataconversion.MediaType;
+import org.infinispan.server.resp.Consumers;
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespErrorUtil;
+import org.infinispan.server.resp.RespRequestHandler;
+import org.infinispan.server.resp.commands.Resp3Command;
+import org.infinispan.server.resp.hll.HyperLogLog;
+
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * The `<code>PFADD key [element [element ...]]</code>` command.
+ * <p>
+ * Adds an element to the HyperLogLog (HLL) structure. The command always verify if is possible to use the HLL structure
+ * with the given <code>key</code>. If the command is invoked without elements, the entry is created.
+ * </p>
+ *
+ * @see <a href="https://redis.io/commands/pfadd/">Redis documentation.</a>
+ * @since 15.0
+ * @author José Bolina
+ */
+public class PFADD extends RespCommand implements Resp3Command {
+
+   public PFADD() {
+      super(-2, 1, 1, 1);
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler, ChannelHandlerContext ctx, List<byte[]> arguments) {
+      byte[] key = arguments.get(0);
+      AdvancedCache<byte[], Object> cache = handler.cache().withMediaType(MediaType.APPLICATION_OCTET_STREAM, null);
```

Comment:
I did this to "cheat" on the cache value type. Otherwise, we would need to use it raw. This is necessary to retrieve the object without trying to convert it to byte[] or other data structures. Do you think there's another approach to that?

Reply:
I think we should evaluate making it so the handler cache already has this set then if it isn't.